test/e2e/destroypod_test.go

```diff
@@ -110,7 +110,7 @@ func TestDestroyPodInflight(t *testing.T) {
 	}
 
 	// The timeout app sleeps for the time passed via the timeout query parameter in milliseconds
-	timeoutRequestDurationInMillis := timeoutRequestDuration / time.Millisecond
+	timeoutRequestDurationInMillis := int64(timeoutRequestDuration / time.Millisecond)
```

Comment:
why this change?

Reply:
Apparently the compiler thought the former is still a `time.Duration` in my case. It was mostly to make the linter happy, I can back it out if it makes you feel uncomfortable.

Reply:
The linter complained that I have a variable of type `time.Duration` and I'm also giving it its unit in the name.

Reply:
Nah that's fine, just wondered why
/hold cancel